docs/tabs.css

```diff
@@ -39,3 +39,10 @@
     border-top: none;
     padding: 6px 12px;
 }
+
+footer {
+  font-size: 0.8em;
+  text-align: center;
+  border-top: 1px solid black;
+  padding: 5px 0;
+}
```

Comment:
This is probably ok for now, but strictly speaking this isn't for the language tabs feature, so maybe it goes in its own css file?

Reply:
That is true, but we have so few CSS rules, I'd rather keep them in a single file. I opted for renaming that file instead.